lib/site-builder.js

```diff
@@ -213,18 +213,36 @@ SiteBuilder.prototype.readOrWriteConfig = function(configExists) {
 
 SiteBuilder.prototype.jekyllBuild = function() {
   var jekyll = config.jekyll;
-  var args = ['build', '--trace', '--destination', this.buildDestination,
-    '--config', '_config.yml,_config_18f_pages.yml'];
+  var args = [];
 
   if (this.usesBundler) {
     jekyll = config.bundler;
-    args = ['exec', 'jekyll'].concat(args);
+    args = ['exec', 'jekyll'];
+  }
+  args = args.concat(['build', '--trace', '--destination']);
+
+  if (!this.hasInternalConfig) {
+    return this.spawn(jekyll, args.concat(
+      [this.buildDestination,
+       '--config', '_config.yml,' + this.opts.pagesConfig]));
```

Comment:
I feel like there could be a way to DRY up the args concat stuff in each of these three `spawn` calls, but I don't have an immediate suggestion for how to do it. Maybe if there were also a `this.opts.configFiles` to avoid hard-coding the various config.yml names? And then some further abstraction possibilities might appear...